packages/calcite-components/src/components/slider/slider.e2e.ts

```diff
@@ -204,6 +204,17 @@ describe("calcite-slider", () => {
     });
   });
 
+  it.skip("step floating point precision", async () => {
```

Comment:
how come this is being added but skipped?

Reply:
Due to #10168. Should I add a comment?

Reply:
The alternative was/is expecting the buggy value (`1.3`) and mentioning in the issue to update the test with the correct value (`1.4`). 

Would you prefer that option? Intentionally committing a bug into our tests would hurt, but I'm sure I could work through it after years of therapy.

Reply:
Its kind of weird to be adding a new test that is skipped. Why add it at all?

Reply:
Yeah maybe just add a comment about the bug/issue.

Reply:
Yeah, I agree it's weird. I wanted to add a test for coverage of #9684, but didn't have time to finish #10168 before the release. The two issues are related to the point where I couldn't repro the floating point precision one without the rounding one occurring too. 

I think adding the test and skipping it is better than not adding it at all or expecting a buggy value. I'll add a comment, thanks.

Reply:
Makes sense! Thanks